docs/react/practical-projects.en-US.md

```diff
@@ -270,6 +270,8 @@ We have completed a simple application, but you may still have lots of questions
 You can:
 
 - Visit [dva official website](https://github.com/dvajs/dva).
-- View all the [API](https://github.com/dvajs/dva#api).
-- View [tutorial](https://github.com/dvajs/dva-docs/blob/master/v1/zh-cn/tutorial/01-%E6%A6%82%E8%A6%81.md), complete a medium application step by step.
-- View examples, such as [dva version of hackernews](https://github.com/dvajs/dva-hackernews).
+- Familiar with the [8 Conpects](https://github.com/dvajs/dva/blob/master/docs/Concepts.md), and know how they are connected together
```

Comment:
`Be` familiar with the [8 `Concepts`](https://github.com/dvajs/dva/blob/master/docs/Concepts.md), and `understand` how they are connected together.